breeze

```diff
@@ -904,7 +904,7 @@ if [[ ${INITIALIZE_LOCAL_VIRTUALENV} == "true" ]]; then
         if [[ ${SYSTEM} == "Darwin" ]]; then
             echo "  brew install sqlite mysql postgresql"
         else
-            echo "  sudo apt-get install openssl sqlite libmysqlclient-dev libmysqld-dev postgresql --confirm"
+            echo "  sudo apt-get install python3-dev python-dev openssl sqlite default-libmysqlclient-dev libmysqld-dev postgresql --confirm"
```

Comment:
Why python-dev and python3-dev?

Reply:
Just in case. I want to backport it to 1.10 so I want it to work there as well. There are still some distros that have python2 as default and it will be like that for a while so it's better to leave it like that just in case.